### PR TITLE
OPENNLP-1446 Investigate why LeskEvaluatorTest and MFSEvaluatorTest fail

### PR DESCRIPTION
**Pull request by mawiesne**
Change
-
- makes WSD evaluator tests run again
- renames existing other WSD tests from "Tester" to "Test" pattern
- restores resources as required originally by the WSD tests
- adds _semcor3.0_ files required for WSD (eval) tests
- fixes minor issues with tests in other modules, see OPENNLP-1532
- resolves _OPENNLP-1446_

Tasks
-
Thank you for contributing to Apache OpenNLP.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with OPENNLP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically main)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via `mvn clean install` at the root opennlp-sandbox folder?
- [x] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file in opennlp-sandbox folder?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found in opennlp-sandbox folder?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by mawiesne** [comment]
Note: These changes also fix massive performance problems due to "slow" code, e.g. by loading resource multiple times for no benefit, as well as by dealing with regex patterns inefficiently. The tests execute much faster compared with the original, previous version.

File: opennlp-wsd/src/test/java/opennlp/tools/disambiguator/WSDTest.java [hunk 1]
```diff
@@ -47,95 +46,62 @@
...
  */
-
-class WSDTester {
-  // TODO write more tests
-  // TODO modify when we fix the parameter model
-
-  static String modelsDir = "src/test/resources/models/";
-  static String trainingDataDirectory = "src/test/resources/supervised/models/";
+// TODO write more tests
+// TODO modify when we fix the parameter model
```
Comment (kinow) [inline opennlp-wsd/src/test/java/opennlp/tools/disambiguator/WSDTest.java:50]: :+1: :point_up:

File: summarizer/src/test/java/opennlp/summarization/lexicalchaining/LexChainTest.java [hunk 2]
```diff
@@ -101,7 +102,7 @@ void testGetRelation() {
       assertEquals(WordRelation.MED_RELATION, rel2.relation);
       assertEquals(WordRelation.MED_RELATION, rel3.relation);
     } catch (Exception e) {
-      e.printStackTrace();
+      fail(e.getLocalizedMessage());
```
Comment (kinow) [inline summarizer/src/test/java/opennlp/summarization/lexicalchaining/LexChainTest.java:105]: :+1: or just let it raise the exception with the whole stack trace.

**Review by kinow — APPROVED** [review]

File: opennlp-wsd/src/test/resources/semcor3.0/LICENSE
Comment (rzo1) [inline opennlp-wsd/src/test/resources/semcor3.0/LICENSE:1]: Given that we are in the sandbox and after reviewing https://www.apache.org/legal/resolved.html , I think, that it is ok to include the test resources here. If we ever migrate it to the core project, we might want to put that resources inside the eval dataset, so the source does not distribute this dataset.

**Review by rzo1 — APPROVED** [review]

**Pull request merged**